source/interface/azure_iot_mqtt.h

```diff
@@ -0,0 +1,380 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_mqtt.h
+ *
+ * @brief Definition for the Azure IoT MQTT.
+ *
+ */
+
+#ifndef AZURE_IOT_MQTT_H
+#define AZURE_IOT_MQTT_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+#if defined( __cplusplus ) || ( defined( __STDC_VERSION__ ) && ( __STDC_VERSION__ >= 199901L ) )
+    #include <stdbool.h>
+#elif !defined( bool ) && !defined( false ) && !defined( true )
+    #define bool     int8_t
+    #define false    ( int8_t ) 0
+    #define true     ( int8_t ) 1
+#endif
+
+#include "azure_iot_mqtt_port.h"
+#include "azure_iot_transport_interface.h"
+
+#define azureiotmqttPACKET_TYPE_CONNECT        ( ( uint8_t ) 0x10U )  /** @brief CONNECT (client-to-server). */
```

Comment:
We can't make these an enum with assigned values?

Reply:
Lets keep it align with coreMQTT

---
In reply to: [598024864](https://github.com/Azure/azure-iot-middleware-freertos/pull/44#discussion_r598024864) [](ancestors = 598024864)